infra/cifuzz/actions/entrypoint.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Builds and runs specific OSS-Fuzz project's fuzzers for CI tools."""
+
+import os
+import subprocess
+
+
+def main():
+  """Runs OSS-Fuzz project's fuzzers for CI tools."""
+  project_name = os.environ['OSS_FUZZ_PROJECT_NAME']
+  repo_name = os.environ['GITHUB_REPOSITORY'].rsplit('/', 1)[-1]
+  commit_sha = os.environ['GITHUB_SHA']
```

Comment:
nit: `GITHUB_SHA` is not a good name IMO, use `GIT_COMMIT` or `COMMIT_ID`

Reply:
GITHUB_SHA is also set by github actions so I can't change it unfortunately.

Reply:
wow!